README.md

```diff
@@ -1,12 +1,89 @@
 # Overview
 
-<img align="right" width="200" src="./doc/images/Logo.jpg" />
+<img src="./doc/images/Overview ReadMe/PT hero image.png" />
 
 Microsoft PowerToys is a set of utilities for power users to tune and streamline their Windows experience for greater productivity. Inspired by the [Windows 95 era PowerToys project](https://en.wikipedia.org/wiki/Microsoft_PowerToys), this reboot provides power users with ways to squeeze more efficiency out of the Windows 10 shell and customize it for individual workflows.  A great overview of the Windows 95 PowerToys can be found [here](https://socket3.wordpress.com/2016/10/22/using-windows-95-powertoys/).
-
-## Build Status
+<br>
+<br>
+[Download][github-release-link]   |   [Release notes][github-release-link]   |   [Microsoft privacy statement][privacyLink]
+<br>
+<br>
+## Build status
 
 [![Build Status](https://dev.azure.com/ms/PowerToys/_apis/build/status/microsoft.PowerToys?branchName=master)](https://dev.azure.com/ms/PowerToys/_build?definitionId=219)
+<br>
+<br>
+
+## Current PowerToy utilities
+
+### FancyZones
+
+<img align="left" src="./doc/images/Overview ReadMe/FancyZones_small.png" />[FancyZones](/src/modules/fancyzones/) is a window manager that makes it easy to create complex window layouts and quickly position windows into those layouts.
+<br>
+<br>
+<br>
+<br>
+<br>
+
+### Shortcut Guide
+
+<img align="left" src="./doc/images/Overview ReadMe/Shortcut guide_small.png" />[Windows key shortcut guide](/src/modules/shortcut_guide)  appears when a user holds the Windows key down for more than one second and shows the available shortcuts for the current state of the desktop.
+<br>
+<br>
+<br>
+<br>
+<br>
+
+### PowerRename
+
+<img align="left" src="./doc/images/Overview ReadMe/PowerRename_small.PNG" />[PowerRename](/src/modules/powerrename) is a Windows Shell Extension for advanced bulk renaming using search and replace or regular expressions. PowerRename allows simple search and replace or more advanced regular expression matching. While you type in the search and replace input fields, the preview area will show what the items will be renamed to. PowerRename then calls into the Windows Explorer file operations engine to perform the rename. This has the benefit of allowing the rename operation to be undone after PowerRename exits.
+
+This code is based on [Chris Davis's SmartRename](https://github.com/chrdavis/SmartRename).
+<br>
+<br>
+
+### File Explorer (Preview Panes)
+
+<img align="left" src="./doc/images/Overview ReadMe/PowerPreview_small.PNG" />[File Explorer](/src/modules/previewpane) add-ons right now are just limited to Preview Pane additions for File Explorer. Preview Pane is an existing feature in the File Explorer.  To enable it, you just click the View tab in the ribbon and then click "Preview Pane".
```

Comment:
Explainer here doesn't follow the same pattern as most of the other modules. It takes an unusual self-deprecating tone based on knowledge of upcoming File Explorer modules. Just explain the modules.